nanohttp/tests/test_validation.py

```diff
@@ -286,3 +287,16 @@ def test_readonly_validator():
         validator(dict(a=None))
     assert '666 a is readonly' == str(ctx.value)
 
+def test_exception_class_in_validator():
+
+    class HTTPCustomException(HTTPKnownStatus):
+        status = '600 Custom Exception'
+
+    validator = Criterion(
+        (True, HTTPCustomException)
+    )
+
+    with pytest.raises(HTTPStatus) as ctx:
+        raise validator.create_exception()
+    assert HTTPCustomException.status == str(ctx.value)
```

Comment:
This assertion is useless. You have to use 'ctx.exception' or 'ctx.error' instead.